app/liquid_tags/forem_tag.rb

```diff
@@ -20,6 +20,7 @@ def self.process_input(input)
   def self.determine_klass(link)
     return ListingTag if link.start_with?("#{URL.url}/listings/")
     return TagTag if link.start_with?("#{URL.url}/t/")
+    return CommentTag if link.include?("/comment/")
```

Comment:
I started to recommend a change here for consistency with the lines above it but then realized that you didn't do it specifically because it requires a dynamic path segment before the `/comment/`. I probably would've done it this way, too, unless I wanted to get clever with a regex (spoiler: I would not want to do that).

I don't know if there's a way to make that clearer but also I don't know if we need to. This code is unlikely to change much unless Forem paths change, which we're unlikely to do because we want to keep URLs working. The only other reason I can see this changing is if we revamp the entire concept of embedded content _again_.

![Please no!](https://c.tenor.com/2Qrxb60EYFoAAAAM/please-no-michael-scott.gif)

Reply:
@jgaskins LOL I thought about getting clever with regex too! I also thought about if someone's username was `comment` (in the which case their posts would render the CommentTag) but quickly ascertained that `comment` was among `ReservedWords.all` from the User model, and thus forbidden as a username.

I think this implementation is fine. 😅